tests/FakeItEasy.Tests.Approval/ApprovedApi/FakeItEasy/net40.approved.txt

```diff
@@ -344,10 +344,8 @@ namespace FakeItEasy.Configuration
 {
     public class ArgumentCollection : System.Collections.Generic.IEnumerable<object?>, System.Collections.IEnumerable
     {
-        [System.Runtime.CompilerServices.IndexerName("Item")]
-        public System.Collections.Generic.IEnumerable<string> Item { get; }
-        [System.Runtime.CompilerServices.IndexerName("Item")]
-        public int Item { get; }
+        public System.Collections.Generic.IEnumerable<string> ArgumentNames { get; }
+        public int Count { get; }

```

Comment:
Where did this change come from? It doesn't seem related to the testing libraries...

Reply:
Doh! Somehow I missed that change. No, you can leave it here.